src/preferencesControl.ts

```diff
@@ -147,6 +147,12 @@ export class PreferencesControl {
     public hidePreferencesDialog(): void {
         let cookieModal = document.getElementsByClassName(styles.cookieModal)[0];
         this.containerElement.removeChild(cookieModal);
+
+        let cookieInfo = document.getElementsByClassName(styles.bannerButton)[1];
```

Comment:
This way of finding elements is dangerous. This `[1]` wouldn't work if there will be another button or the order is changed or many other things could happen. There are 3 possible ways to do it:
1. Write unit test
2. Give more specific CSS class just for this button
3. Give unique ID.

#1 is the most preferred. #3 is the least preferred.